### PR TITLE
Tabs

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/react_tabs/)

### CODE REVIEW

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  return (
+    <ul className="tabs">
+      {tabs.map(tab => (
+        <li key={tab.id} className="tabs__tab">
+          <a
+            href="/"
+            className={classNames('tabs__link', { 'tabs__link--selected': selectedTabId === tab.id })}
```
Comment (Vitaliy-0) [inline src/components/Tabs/Tabs.tsx:18]:
```suggestion
            className={classNames('tabs__link',
                   { 'tabs__link--selected': selectedTabId === tab.id })}
```

**Review by Vitaliy-0 — APPROVED** [review]
Good job!

File: src/App.tsx [hunk 2]
```diff
@@ -1,18 +1,50 @@
...
+  setSelectedTab = (tabId: string) => {
+    this.setState({ selectedTab: tabs.find(tab => tab.id === tabId) || tabs[0] });
+  };
+
+  render() {
+    return (
+      <div className="app">
+        <h1 className="app__title">
+          Selected tab is&nbsp;
+          {this.state.selectedTab.title}
```
Comment (illiarerun) [inline src/App.tsx:31]: use template string

File: src/components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            onClick={(event) => {
+              event.preventDefault();
+              onTabSelected(tab.id);
+            }}
+          >
+            {tab.title}
+          </a>
```
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:25]: Make it a button

File: src/components/Tabs/Tabs.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  return (
+    <ul className="tabs">
+      {tabs.map(tab => (
+        <li key={tab.id} className="tabs__tab">
+          <a
+            href="/"
+            className={classNames('tabs__link', { 'tabs__link--selected': selectedTabId === tab.id })}
+            onClick={(event) => {
+              event.preventDefault();
```
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:20]: So you wouldn't need it

File: src/components/Tabs/Tabs.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+              onTabSelected(tab.id);
```
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:21]: Also, what if the tab is already selected?

File: src/components/Tabs/Tabs.tsx (hunk 1 above)
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:18]:
```suggestion
            className={classNames(
              'tabs__link',
              { 'tabs__link--selected': selectedTabId === tab.id }
            )}
```

File: src/App.tsx [hunk 6]
```diff
@@ -1,18 +1,50 @@
...
+type State = {
+  selectedTab: Tab,
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedTab: tabs[0],
+  };
+
+  setSelectedTab = (tabId: string) => {
```
Comment (illiarerun) [inline src/App.tsx:22]: You can pass the whole tab to avoid using `find` in your setState

**Review by illiarerun — CHANGES_REQUESTED** [review]
GJ 🔥 Let's work on fixes:

**Review by illiarerun — APPROVED** [review]
GJ 🔥

**Pull request closed without merging**